Responmessenger.css

```
@@ -0,0 +1,191 @@
+html,body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
+}
+.cover{
+    max-width: 1024px;
+    margin:0 auto;
+}
+
+.menu{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    height: 80px;
+    position: sticky;
+    top: 0;
+    border-bottom: 1px solid lightgray;
+    background-color: #fafafa;
+    opacity: 0.9;
+}
+
+.logocontainer{
```

would be great if you could be consistent in using name convention. 
```suggestion
.logoContainer{
```

---

yes sir I am gonna care to be name convention